.github/workflows/default.yml

```diff
@@ -38,13 +38,13 @@ jobs:
         run: cargo install zepter --locked --version 0.11.0 -q -f --no-default-features && zepter --version
 
       - name: Check toml formatting
-        run: zepter format features --check
```

Comment:
Why are we doing this? @Leouarz

Reply:
Zepter command was not introduced by me. Now it checks the specified toml file, i removed check since it's implicit now.

Reply:
The check is now implicit 
![image](https://github.com/availproject/avail/assets/15839293/0d6bb9be-cc35-47c7-90d7-b86b0da42db6)

To fix you have to add --fix

Reply:
Ok, got it